content/zh/docs/application-store/built-in-apps/harbor-app.md

```diff
@@ -20,17 +20,17 @@ weight: 14220
 
 1. 在 `demo-project` 项目的**概览**页面，点击左上角的**应用商店**。
 
-   ![应用商店](/images/docs/zh-cn/appstore/built-in-apps/deploy-harbor-on-ks/app-store.PNG)
+   ![应用商店](/images/docs/zh-cn/appstore/built-in-apps/deploy-harbor-on-ks/app-store.png)
```

Comment:
Check all the images in your local build to see if they are dead.

Reply:
All good.